kivy/__init__.py

```diff
@@ -251,6 +252,16 @@ def kivy_usage():
 #: Kivy user extensions directory
 kivy_userexts_dir = ''
 
+# if there are deps, import them so they can do their magic.
+try:
+    import kivy.deps
```

Comment:
I think you should separate the kivy.deps and the exception, in order to know if the exception came from kivy.deps directly, or one of the modules in the kivy.deps.__path__